#
# The Net-SNMP libraries are built without -zdefs and are missing a large
# number of dependency specifications (DT_NEEDED).  This makes it
# impossible to build plugin modules against these libraries using
# -zdefs, which should work. One such example is FMA in ON gate. 
#
--- net-snmp-5.7.2/ltmain.sh	Sun Dec 30 23:42:05 2012
+++ net-snmp-5.7.2/ltmain.sh	Sun Dec 30 23:49:01 2012
@@ -5406,7 +5406,9 @@
 	  if test "$prev" = rpath; then
 	    case "$rpath " in
 	    *" $arg "*) ;;
-	    *) func_append rpath " $arg" ;;
+	    *) func_append rpath " $arg"
+		# This is make sure only $libdir path is encoded in RUNPATH
+			sma_rpath="$rpath" ;;
 	    esac
 	  else
 	    case "$xrpath " in
@@ -8046,7 +8048,7 @@
 		fi
 	      else
 		eval flag=\"$hardcode_libdir_flag_spec\"
-		func_append dep_rpath " $flag"
+#func_append dep_rpath " $flag"
 	      fi
 	    elif test -n "$runpath_var"; then
 	      case "$perm_rpath " in
